yarn-project/kv-store/src/lmdb/store.ts

```diff
@@ -144,23 +144,35 @@ export class AztecLmdbStore implements AztecKVStore {
   }
 
   /**
-   * Clears all entries in the store
+   * Clears all entries in the store & sub DBs.
    */
   async clear() {
+    await this.#data.clearAsync();
+    await this.#multiMapData.clearAsync();
     await this.#rootDb.clearAsync();
   }
 
+  /**
+   * Drops the database & sub DBs.
+   */
+  async drop() {
+    await this.#data.drop();
+    await this.#multiMapData.drop();
+    await this.#rootDb.drop();
+  }
+
   /**
    * Close the database. Note, once this is closed we can no longer interact with the DB.
-   * Closing the root DB also closes child DBs.
    */
   async close() {
+    await this.#data.close();
```

Comment:
Did closing the root DB not close these? Docs seemed to say it would

Reply:
ChatGPT told me it didn't and indeed, as I was looking into objects that were in-memory, there were 3 per store, (rootDB, data, data_dup_sort). with previous code, data & data_dup_sort still had `status: 'open'` until I added all `close` statements.